packages/web/components/Quest/QuestDetails.tsx

```diff
@@ -11,6 +11,7 @@ import {
   Text,
   VStack,
 } from '@metafam/ds';
+import { Prose } from '@nikolovlazar/chakra-ui-prose';
```

Comment:
you can probably export `Prose` from design system and then you only need the dependency to `chakra-ui-prose` only over there and remove from `web`

Reply:
Done in the next update